main.tf

```diff
@@ -12,6 +12,28 @@ provider "aws" {
   profile = "codeforpoznan"
 }
 
+data "aws_iam_policy_document" "codeforpoznan_public_policy_document" {
+  version = "2012-10-17"
+
+  statement {
+    sid    = "PublicListBucket"
+    effect = "Allow"
+    principals {
```

Comment:
why these two statements differ in respect to `principals`?

Reply:
missed it, fixed